lisa/sut_orchestrator/azure/features.py

```diff
@@ -1913,12 +1912,7 @@ def enabled(self) -> bool:
 class ArchitectureSettings(schema.FeatureSettings):
     type: str = "Architecture"
     # Architecture in hyper-v
-    arch: str = field(
-        default="x64",
-        metadata=field_metadata(
-            validate=validate.OneOf(["x64", "Arm64"]),
-        ),
-    )
+    arch: str = "x64"
```

Comment:
Can we keep the validation? The value should be normalized before set to this field. So it wouldn’t bring other failures later.

Reply:
I tried with validation, but still encounter this issue. Where should I add the normalized value?

Reply:
Is it possible there is an invalid value? What’s the value before assigned?